packages/examples/CipEthernetIP/README.md

```diff
@@ -0,0 +1,22 @@
+# Using CIP EtherNet Industrial Protocol as Transport in LIOTA
+
+LIOTA offers CIP EtherNet Industrial protocol as transport at Device end via [EtherNetIP_DeviceComms](https://github.com/lucifercr07/liota/blob/can_bus/liota/device_comms/canbus_device_comms.py)
```

Comment:
EtherNetIP_DeviceComms? Correct the name.

Reply:
ok